test/controllers/LanguageControllerSpec.scala

```diff
@@ -18,13 +18,12 @@ package controllers
 
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
-import uk.gov.hmrc.play.binders.ContinueUrl
 import util.BaseSpec
 
 class LanguageControllerSpec extends BaseSpec {
 
   trait LocalSetup {
-    val c = app.injector.instanceOf[LanguageController]
+    val c = app.injector.instanceOf[LanguageSwitchController]
```

Comment:
Can we rename this val to make more readable

Reply:
"c" changed to "langSwitchController"